cmd/minishift/cmd/config/config.go

```diff
@@ -100,6 +100,14 @@ var settings []Setting = []Setting{
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
+		name: "hostfolders-automount",
```

Comment:
For the homefolder I can understand this to be on global level...  And I would prefer to not have this in the future
```
$ minishift hostfolder add Users --global --username="dsd" --password="fdfs" ...
$ minishift hostfolder add Disks --instance=minishift --username="dsd" --password="fdfs" ...
```

Would like feedback...